.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+## FreeCodeCamp - Wiki Pull Request template
+
+Make sure your contribution addresses the points mentioned below:
+
+Filenames:
+- Your filename shoud NOT contain any special chars or spaces.
+- Use only "-" to separate words in the filename.
+- Filenames should have extention ".md"
+
+Formatting:
+- Ensure that you use markdown syntax for articles.
+- Checkout the guides and cheetsheets for reference:
+1. https://guides.github.com/features/mastering-markdown/
+2. https://help.github.com/articles/working-with-advanced-formatting/
+
+Commits and Squashing:
+- Your PR/Submission should have only one commit.
+
+- For multiple commits follow the below guide to squash.
+  https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one
+
+- Only in the case, when you are adding a wiki article from browser, we will squash it for you.
```

Comment:
Add somethign along these lines, if you are new to contributing then you can use the browser,and we will squash for you if requested. However, for more control over the pull request we request that you use the local method and squash yourself. 

The links are here https://github.com/FreeCodeCamp/FreeCodeCamp/wiki feel free to use them and remaind them of the points from the homepage.